docs/start/overview.md

```diff
@@ -8,21 +8,100 @@ activePill: overview
 # What Is Shield?
 
 Shield is the mortar between the brickwork of your service oriented architecture, the glue that fits services smoothly 
-together. 
+together. It gathers multiple services into a single domain and adds a production ready varnish on top in the form of
+HTTP middleware.
 
-API Gateway definition
+Shield is ostensibly another implementation of the API Gateway pattern, but provides a unique combination of configurable
+middleware, upstream service aggregation, and intelligent request routing.
 
-1. Router
-1. Load Balancer
-1. Circuit Breaker
-1. Middleware Engine
+For those unfamiliar with the API Gateway pattern, the gateway acts as the primary entrypoint into a microservice 
+ecosystem.  Depending on how it's configured, both internal and external clients may send requests to the gateway for 
+delivery to the appropriate upstream service.
 
-Microservice Integration
+## Shield: Service Discovery 
 
-1. Service Discovery
-1. Endpoint Discovery (Swagger)
+To make the life of an infrastructure engineer as easy as possible, your tooling must be as automated as possible.  
+Shield integrates with existing service discovery mechanisms to detect new deployments and scaling events and automatically
+start sending traffic to them.
 
-Production Ready
+<img src="{{ site.baseurl }}/assets/images/service_discovery.gif" class="img-responsive">
+
+As these new hosts come online, Shield will retrieve their [Swagger Documentation](http://swagger.io) that describes 
+what endpoints they are capable of serving.  Shield will intelligently route traffic to that host if it's actually 
+capable of handling the request.
+
+## Shield: The Router
+
+When Shield is in front of multiple unique services, it needs to know where to forward the request on to.  To
+solve this routing problem, Shield builds a routing table for all the upstream endpoints that were discovered
+during the Service Discovery phase.  The routing table then maps a request to a service by looking at the HTTP method, 
+path, and content type of the request and desired response.
+
+<img src="{{ site.baseurl }}/assets/images/routing.gif" class="img-responsive">
+
+This routing is fast, on the order of 0.01ms, even when choosing between hundreds of endpoints exposed by the upstream
+services.  
+
+Shield is smart when multiple upstream endpoints could be used to satisfy the request, and uses the [most
+specific endpoint]({{ site.baseurl }}/start/lifecycle/endpoint_sorting) possible when deciding which upstream to proxy
+to.
+
+## Shield: The Load Balancer
+
+When Shield is in front of multiple instances of the same service, it will load balance requests across the different 
```

Comment:
A reader with no context might be confused by the narrative of this paragraph, I think. In particular, "load balancing occurs on a per endpoint basis" comes across as a plot twist.

To make that clear from the beginning, we could:
1. start this paragraph mentioning that Shield load-balances on a per-endpoint basis
2. go on to describe the ramifications: you get load-balancing "for free" whether you have multiple instances of the same service, or different services advertising the same endpoint.